docs/API_AND_INTEGRATION_GUIDE.md

```
@@ -231,6 +265,137 @@
+```json
+{
+  "total_kills": 245,
+  "systems_processed": 3,
+  "errors": []  // Any errors encountered
+}
+```
+
+#### Rate Limiting
+
+The preload system includes built-in rate limiting to prevent API blocking:
+- **zkillboard**: 10 requests per minute
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)